internal/commands/inspect.go

```diff
@@ -17,7 +17,7 @@ func inspectCmd(dockerCli command.Cli) *cobra.Command {
 		Short: "Shows metadata, parameters and a summary of the compose file for a given application",
 		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			muteDockerCli(dockerCli)
+			defer muteDockerCli(dockerCli)()
```

Comment:
why has it changed ?

Reply:
I don't know, we'll need to ask @silvin-lubecki when he's back on Monday. We can follow up with a fix for this if needed.

Reply:
The `muteDockerCli` now returns an `undo` function closure, so this is muting at this point and then unmuting (via the deferred call of that closure) on exit from the function.